src/plugins/cordova-plugin-battery-status/sim-host-panels.html

```diff
@@ -1,13 +1,13 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. -->
 
-<cordova-panel id="battery-status" caption="Battery Status" data-loc-id="ac1dbd34">
+<cordova-panel id="battery-status" caption="Battery Status" spoken-text="Battery Status Panel" data-loc-id="ac1dbd34">
     <cordova-group>
         <cordova-panel-row>
-            <cordova-label for="battery-level" label="Battery level" data-loc-id="e0d4e236"></cordova-label>
+            <cordova-label spoken for="battery-level" label="Battery level" data-loc-id="e0d4e236"></cordova-label>
             <cordova-label id="battery-level-label" label="100%" style="margin-right:5px;margin-left:auto;"></cordova-label>
             <input id="battery-level" type="range" value="100" min="0" max="100">
         </cordova-panel-row>
     </cordova-group>
 
-    <cordova-checkbox id="is-plugged" checked="" data-loc-id="10cfe833">Device connected to power</cordova-checkbox>
+    <cordova-checkbox id="is-plugged" checked="" spoken data-loc-id="10cfe833">Device connected to power</cordova-checkbox>
```

Comment:
Why do some checkboxes have the `spoken` attribute and some don't? Why do checkboxes even need this attribute (when would a checkbox ever not need to be spoken)?